### PR TITLE
Pre release cleanup

### PR DESCRIPTION
**Pull request by clapierre**
Here are all my changes that we agreed upon during our call last Friday.

### CODE REVIEW

**Review by gregoriopellegrino — APPROVED** [review]

**Comment by clapierre** [comment]
@mattgarrish Can you do a final review and if you also approve merge this as @avneeshsingh has requested in the email I am forwarding to you now :)

File: UX-Guide-Metadata/principles/index.html [hunk 1]
```diff
@@ -96,7 +96,7 @@
 		<section id="general-overview">
 			<h2>General Overview</h2>
 			<p>This document will help those who wish to provide accessibility metadata directly to users understand how
-				to represent machine readable accessibility metadata in a user friendly User Interface / User Experience
+				to represent machine readable accessibility metadata in a user-friendly User Interface / User Experience
```
Comment (mattgarrish) [inline UX-Guide-Metadata/principles/index.html:99]: machine-readable should be hyphenated, too.

File: UX-Guide-Metadata/principles/index.html [hunk 2]
```diff
@@ -136,7 +136,7 @@ <h2>Discovering Accessible Content</h2>
 			<p>While every user has different accessibility needs, meeting the Web Content Accessibility Guidelines
 				(WCAG) 2.0 requirements, even at level A, is a baseline that ensures a book will be widely accessible.
 				Book providers may therefore wish to create specific search capabilities to permit users to find books
-				that have declared conformance to WCAG at any level. A search filter called Accessible that retrieves all books with metadata that indicates conformance to WCAG Level A, AA and/or AAA.</p>
+				that have declared conformance to WCAG at any level. A search filter called Accessible that retrieves all books with metadata that indicates conformance to WCAG Level A, AA or AAA.</p>
```
Comment (mattgarrish) [inline UX-Guide-Metadata/principles/index.html:139]: Shouldn't Accessible be in quotes and noted as an example of how this should work?

> A search filter called "Accessible", for example, that retrieves...

**Review by mattgarrish — COMMENTED** [review]
Generally looks fine, but if you ask me to review you force me to make comments... 😄

File: UX-Guide-Metadata/principles/index.html (hunk 1 above)
Comment (clapierre) [reply UX-Guide-Metadata/principles/index.html]: Ok just pushed that change :)

File: UX-Guide-Metadata/principles/index.html (hunk 2 above)
Comment (clapierre) [reply UX-Guide-Metadata/principles/index.html]: Done

**Comment by clapierre** [comment]
Ok @mattgarrish Matt all fixed.

**Pull request merged**